synapse/config/server.py

```diff
@@ -24,6 +24,14 @@
 
 logger = logging.Logger(__name__)
 
+# by default, we attempt to listen on both '::' *and* '0.0.0.0' because some OSes
+# (windows? old linux? those where net.ipv6.bindv6only is set?) will only listen
+# on IPv6 when ':' is set.
```

Comment:
`::`.  This applies to Windows and BSD-variants where `net.ipv6.bindv6only` is set (ie. operating systems that do not follow RFC3493). See #2232.

Reply:
Interesting. https://github.com/matrix-org/synapse/pull/4586/files#diff-7604431227e17cc581f690570dfacca6L306 appears to say the opposite for Mac OS (by which I assume it means macOS, or however Apple are spelling it this month) ?